src/main/java/carracing/domain/car/strategy/CarPowerCondition.java

```diff
@@ -0,0 +1,23 @@
+package carracing.domain.car.strategy;
+
+import java.util.Random;
+
+public class CarPowerCondition implements CarCondition {
+
+    private static final int MOVE_POSITION = 4;
+    private int power;
+
+    public CarPowerCondition() {
+        this.power = getPower();
+    }
+
+    public int getPower() {
+        Random random = new Random();
+        return random.nextInt(10);
+    }
+
+    @Override
+    public boolean isMovable() {
+        return power >= MOVE_POSITION;
```

Comment:
```suggestion
        return random.nextInt(10) >= MOVE_POSITION;
```
CarPowerCondition 클래스가 상태값을 가지고 있어야 할 이유는 없어보여요. :) 
위와 같이 구현해보는건 어떠신가요 ?

Reply:
적용하였습니다!